src/hobbygroups/components/HobbyCard/Divider/index.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+import style from './divider.less';
+
+export const Divider = () => <div className={style.divider} />;
```

Comment:
Does this really need it's own file? I think it is so small that it is unnecessary, but 🤷‍♂

Reply:
I is a larger discussion, but it is a way of writing components more like atoms (see Atomic Design).

The divider is a small and reusable component that can be used anywhere, so why should we have to make it again everywhere it should be used?